src/Symfony/Bundle/DependencyInjection/ApiPlatformExtension.php

```diff
@@ -476,6 +480,15 @@ private function registerGraphQlConfiguration(ContainerBuilder $container, array
 
         $loader->load('graphql.xml');
 
+        // @phpstan-ignore-next-line We ignore the next line because we specify to phpstan to use the container of the test env cache and in test the parameter kernel.bundles always contains the key TwigBundle
```

Comment:
```suggestion
        // @phpstan-ignore-next-line because PHPStan uses the container of the test env cache and in test the parameter kernel.bundles always contains the key TwigBundle
```